pynest/nest/tests/test_refractory.py

```diff
@@ -211,6 +213,11 @@ def test_refractory_time(self):
             # Get and compare t_ref
             t_ref_sim = self.compute_reftime(model, sd, vm, neuron)
 
+            # For models that first clamp the membrane potential
+            # at a higher value (see e.g. aeif_cbvg_2010)
+            if "t_clamp" in nest.GetDefaults(model):
```

Comment:
check for existence of a parameter with a particular name might lead to problems with new models in the future. maybe better maintain a list of models where this should be done as in the other test?

Reply:
Agreed. We changed it with 461bee2